### PR TITLE
enhance importing OpenAPI doc

### PR DESCRIPTION
**Pull request by eguzki**
Explained how 3scale is configured from the OpenAPI document. Which parts of the OpenAPI document are used, and how, to configure 3scale.

Also added documentation about the minimum OpenAPI Document requried to configure a working 3scale product.

https://issues.redhat.com/browse/THREESCALE-4712

### CODE REVIEW

File: docs/openapi.md [hunk 1]
```diff
@@ -1,36 +1,251 @@
...
+```
+
+When OpenAPI does not specify any security requirements:
+* The product is considered as an "Open API"
+* `default_credentials` 3scale policy will be added (also called as `anonymous_policy`)
+* The command option `--default-credentials-userkey` is required and the command will fail if not provided.
+
+#### ActiveDocs
+
+A 3scale ActiveDoc will be created (or updated if previously existed).
```
Comment (miguelsorianod) [inline docs/openapi.md:169]: Does that mean that if it previously existed it can alter it in some way? is this controllable? what can potentially differ?

**Review by miguelsorianod — APPROVED** [review]
Looks good to me :+1:  I just left a minor question

File: docs/openapi.md (hunk 1 above)
Comment (eguzki) [reply docs/openapi.md:169]: For example, when you add/delete/update an operation, the activedoc object is updated accordingly.

Comment (miguelsorianod) [reply docs/openapi.md:169]: :+1:

**Pull request merged**